### PR TITLE
Update vision model check in utils.ts

### PR DESCRIPTION
**Pull request by wolf-joe**
compatible with openrouter model format, see
- https://openrouter.ai/models/google/gemini-flash-1.5
- https://openrouter.ai/models/google/gemini-pro-1.5

## Summary by CodeRabbit

- **New Features**
  - Added support for two new model versions: "gemini-pro-1.5" and "gemini-flash-1.5".

### CODE REVIEW

File: app/utils.ts [hunk 1]
```diff
@@ -255,6 +255,8 @@ export function isVisionModel(model: string) {
     "claude-3",
     "gemini-1.5-pro",
     "gemini-1.5-flash",
```
Comment (Dogtiti) [inline app/utils.ts:257]: "gemini-1.5-pro",
 "gemini-1.5-flash"
 
Is it the same as above?

**Review by Dogtiti — CHANGES_REQUESTED** [review]

File: app/utils.ts (hunk 1 above)
Comment (wolf-joe) [reply app/utils.ts:257]: "gemini-1.5-pro" and "gemini-pro-1.5" are the same model, just named differently by different vendors.

Comment (wolf-joe) [reply app/utils.ts:257]: see https://openrouter.ai/models/google/gemini-pro-1.5

**Comment by Dogtiti** [comment]
I believe that using a custom model feature is more appropriate here. We can't keep adding identical models indefinitely if they are just named differently by different providers.

**Comment by wolf-joe** [comment]
> I believe that using a custom model feature is more appropriate here. We can't keep adding identical models indefinitely if they are just named differently by different providers.

The issue is that when the model is set to `gemini-pro-1.5`, even if it is displayed as `gemini-1.5-pro` through a custom model, the image upload feature cannot be used.

**Pull request closed without merging**